packages/reactive_ble_mobile/ios/Classes/Prelude/CoreBluetooth+Extensions.swift

```diff
@@ -0,0 +1,32 @@
+import class CoreBluetooth.CBCharacteristic
+import class CoreBluetooth.CBService
+import class CoreBluetooth.CBPeripheral
+
+/// Extensions below are supposed to backport API breaking changes in CoreBluetooth
+/// framework which Apple introduced with Xcode 13:
+///
+/// ```
+/// + weak var peripheral: CBPeripheral? { get }
+/// - unowned(unsafe) var peripheral: CBPeripheral { get }
+/// ```
+///
+/// Thus this code shadows original property declarations in CoreBluetooth and changes their semantics
+/// from `unsafe non-optional` to `weak optional` to mimic Xcode 13 behaviour.
+///
+/// - Note: This code compiles only when using Xcode 12 and below.
+/// - SeeAlso: https://forums.swift.org/t/is-unowned-unsafe-t-weak-t-a-breaking-change/49917
```

Comment:
Thanks for adding this explanation, it looks good.

One nit-pick, though: a doc-comment (`///`) is associated with the declaration following it, but your explanation is not (instead it "belong" to the file scope: "Extensions below ..."), therefore I'd suggest to

- [ ] use a normal comment (`//`) in this case and separate it with an empty line from any following construct.

I hope my original wording ("comment or a doc-comment") was not too confusing. What I meant is that either would satisfy me, but you should choose the type that suits your particular explanation best. In this case, I believe, a normal comment should work best.

Reply:
I've decided to move comment to the top since it's related to the whole file

Reply:
That is fine. But a doc-comment should be attached to a declaration, usually. Then it will be rendered in the quick-help panel and can be included in the generated HTML (or whatever) documentation, if anybody cares to generate it.

I'm not sure this arrangement associates the explanation with any reasonable entity (a declaration or the file), therefore I'm suggesting to make it a normal comment (unless you checked it and can confirm that the doc-comment is being associated with something reasonable).

Reply:
@ky1vstar Perfect, thanks you.